examples/source_separation/test/__init__.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+
+_THIS_DIR = os.path.abspath(os.path.dirname(__file__))
+_METRICS_REFERENCE = os.path.join(_THIS_DIR, "metrics", "reference.py")
+
+
+sys.path.append(os.path.join(_THIS_DIR, "..", "..", "..", "test"))
+
+
+def _download_metrics_reference():
+    import requests
+
+    url = "https://raw.githubusercontent.com/naplab/Conv-TasNet/e66d82a8f956a69749ec8a4ae382217faa097c5c/utility/sdr.py"
```

Comment:
I'd move this into the third_party folder as a subfolder named e.g. Conv-TasNet or whatever you prefer. Just make sure you provide proper reference

Reply:
Moved the test code to the unit test and pulled in the reference implementation as `test/torchaudio_unittest/example/souce_sepration/sdr_reference.py.`